packages/client/components/MeetingTopBar.tsx

```diff
@@ -47,7 +47,7 @@ const PrimaryActionBlock = styled('div')({
   display: 'flex'
 })
 
-const AvatarGroupBlock = styled('div')({
+export const AvatarGroupBlock = styled('div')({
```

Comment:
+1 now that we use this section not only for avatars, should we figure out a better name for it? not a big deal but it'd be nice

Reply:
Definitely agree.  Renamed to `IconGroupBlock` (block of avatar icons, options icon, etc.), but open to suggestions!

Reply:
i don't have any better suggestions so I'll remain silent, `IconGroupBlock` sounds much better!